data_steward/resource_files/schemas/pipeline_tables/_deid_questionnaire_response_map.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I would suggest storing this schema as an `schemas/internal/lookup_tables` or `schemas/internal/deid`.  Especially since we aren't storing this to the `pipeline_tables` dataset now.